src/pages/guides/getting_started/code-playground-workflow.md

```diff
@@ -0,0 +1,188 @@
+2. Toggle on **Code Playground** at the bottom of the panel; it will open with the last session you were working on.
+
+To browse your saved sessions:
+
+1. Click the **More** button in the top right corner of the playground window
+2. Select the **Manage Sessions** item in the dropdown menu
+3. You'll see a list of your saved sessions; click on one to **select it** and click the **Open** button to resume working on it.
+
+![Code Playground Add-on Mode](./img/playground--sessions.gif)
+
+<InlineAlert slots="header, text1" variant="info"/>
+
+#### Accessing "Your add-ons" Page
+
+- **Without a document open:** Click the **Add-ons** button in the left rail, then click the **Add-on development** toggle in the top right
+- **With a document open:** Click the **Add-ons** button in the left rail, select the **Your add-ons** tab, then click the "Manage add-ons" link in the Add-on Testing section
+
+## Downloading Your Code
+
+### How to Download
+
+Downloading your code is a great way to save your work and continue working on it locally in your CLI.
+
+To download your code:
+
+1. Click the **More** button in the top right corner of the playground window
+2. Click the **Download** button
+3. This will download a zip file containing your code
+
+![Code Playground Download code](./img/playground--download.png)
+
+### Folder Structure
+
+The downloaded zip file will contain a folder with the following structure:
+
+- add-on folder
+- `script.js` file
+
+![Downloaded Folder Structure](./img/downloaded-folder-structure.png)
+
+### Running Downloaded Code
+
+You can run your add-on folder as a local add-on project in your CLI by following the steps in the [Quickstart Guide](../getting_started/hello-world.md). There is a readme file in the add-on folder that will guide you through the process as well.
+
+**Note:** You cannot run the `script.js` file alone.
+
+## Productivity Tips
+
+### Development Workflow
+
+1. **Start with Script Mode**: Use Script Mode to prototype and test your core functionality.
+2. **Iterate Quickly**: Use keyboard shortcuts to save and run code frequently.
+3. **Test Incrementally**: Run your code often to catch issues early.
+4. **Transition to Add-on Mode**: Once your logic is working, move to Add-on Mode to build the UI.
+5. **Download for Local Development**: Use the download feature to continue development locally.
+
+## Troubleshooting Common Issues
+
+### Code Not Running
+
+- Check for syntax errors in your code.
+- Ensure you're using the correct mode for your use case
+- Verify that all required APIs are properly imported.
+- Check the browser console for error messages.
+
+### Session Not Saving
+
+- The Code Playground supports up to 2MB of storage per session, and 60 sessions per user. Delete unused sessions to free up space.
+- Code is not saved automatically—read the [Saving Your Work](#saving-your-work) section for more details.
+- Make sure you're not in incognito/private browsing mode.
```

Comment:
We can remove this now. Session limits or network issues are the only two reasons a session will not be backed up.